### ISSUE TITLE
Only the _id is displayed

### ISSUE THREAD

**Issue by nicothed** [mingo-app/mingo#148]
I have a very simple collection (see example doc below), and only the _id is displayed, not the other fields:
![image](https://user-images.githubusercontent.com/9327873/73478025-4e239700-4363-11ea-9edd-1fc2033db72b.png)

{
  "_id": "5e2fa7f4ebb9ea768b857080",
  "w": "wine",
  "c": 1241842,
  "l": 4
}

**Comment by nicothed** [comment]
More generally, I don't understand why only a small subset of fields are displayed by default (and with no way to display them all?). 
This is very unsettling and a complete deal-breaker for me.

**Comment by tothradoslav** [comment]
You’ve got a point here. Let me explain the current logic and we can improve upon it. 

Basically, Mingo tries to show a couple od “important” columns by default and considers indexed fields as important. Your collection probably has no indexes. 

We’ll change this so that when no or few indexed fields are available, mingo will show the first couple. 

Showing ALL fields would be great in the case of simple collections as yours but impossible (and useless) in large collections with deep doc structure. 

What do you think?

**Comment by tothradoslav** [comment]
Also, you can add/remove/customize and rearrange the columns as you wish and mingo will remember your settings for each collection separately.

**Comment by nicothed** [comment]
I see, thank you for the explanation. 
Well, I really like Studio *T's behavior: display automatically ALL 1st level fields in the documents displayed. So it's a bit of a "structure analysis" as well. 
Honestly, I can't imagine any other behavior... - maybe it's because I've used Studio 3T too much? :)

**Comment by tothradoslav** [comment]
Mingo is doing a structure analysis and using it. I have changed the behavior so that when there aren't enough indexed fields, we fill up the table with more columns from the documents. 

I have also added "Pin all top-level fields" and "Pin important fields" in the options in the right: https://grabber.app/o2qkvi 

And yes, we've had users reporting similar issues at first (being used to robomongo or studio 3t), but once you understand the new logic, it's much more comprihensive and userfriendly. 

(all this will be available in next release, will close issue once released)
